CHANGELOG.md

```diff
@@ -8,6 +8,23 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [3.0.1] - 2023-09-13
+
+### Added
+
+- In the WordPressCS v3.0.0, the functionality of the `WordPress.Security.EscapeOutput` sniff was changed to flag the unescaped parameters passed to exceptions in the `throw` statements. This sniff now has more modular error codes based on whether the error happened while checking the exceptions or while outputting unescaped content. The error code has only changed for the exceptions case (now `ExceptionNotEscaped`), and not the rest of the sniff (still `OutputNotEscaped`). This will enable the users to silence the error based on the error code if they need to.
+- Props [@anomiex].
```

Comment:
```suggestion
- In the WordPressCS v3.0.0, the functionality of the `WordPress.Security.EscapeOutput` sniff was updated to also flag unescaped parameters passed to exceptions created in `throw` statements. This specific error now has a separate error code: `ExceptionNotEscaped`. This will allow users to ignore/exclude that specific error based on the error code if they need to. Props [@anomiex].
    The error code(s) for other escaping issues flagged by the sniff remain unchanged.
```

Reasons for my change suggestions:
* The sniff has more (unchanged) error codes, not just `OutputNotEscaped`.
* The other error codes have _not_ been made more modular, this one has just been split off.
* "ignore/exclude" references the terms used in _ignore_ annotations/ruleset _excludes_, while "silence" does not.